zips/zip-0230.rst

```diff
@@ -349,13 +350,15 @@ OrchardZSA Asset Burn Description
 
 An OrchardZSA Asset Burn description is encoded in a transaction as an instance of an ``AssetBurn`` type:
 
-+-------+---------------+-----------------------------+--------------------------------------------------------------------------------------------------------------------+
-| Bytes | Name          | Data Type                   | Description                                                                                                        |
-+=======+===============+=============================+====================================================================================================================+
-| 32    | ``AssetBase`` | ``byte[32]``                | For the OrchardZSA protocol, this is the encoding of the Asset Base :math:`\mathsf{AssetBase^{Orchard}}\!`.        |
-+-------+---------------+-----------------------------+--------------------------------------------------------------------------------------------------------------------+
-| 8     | ``valueBurn`` | ``uint64``                  | The amount being burnt. The value is checked by consensus to be non-zero.                                          |
-+-------+---------------+-----------------------------+--------------------------------------------------------------------------------------------------------------------+
++-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
+| Bytes                       | Name                         | Data Type                                      | Description                                                         |
++=============================+==============================+================================================+=====================================================================+
+| 32                          |``AssetBase``                 |``byte[32]``                                    |For the OrchardZSA protocol, this is the encoding of the Asset Base  |
+|                             |                              |                                                |:math:`\mathsf{AssetBase^{Orchard}}\!`.                              |
++-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
+| 8                           |``valueBurn``                 |``uint64``                                      |The amount being burnt. The value is checked by consensus to be      |
```

Comment:
Given that this *is* a burn, in contrast to the ZSF deposit, I think it's important to have different naming.